### ISSUE TITLE
device selector containing interface not known to host

### ISSUE THREAD

**Issue by zolug** [networkservicemesh/cmd-forwarder-vpp#658]
A [device selector](https://github.com/networkservicemesh/cmd-forwarder-vpp/blob/main/internal/config/config.go#L55) file might contain devices that do not exist on the host when the forwarder is starting.

Devices are stored in a map, thus it's non-deterministic when [InitLinks()](https://github.com/networkservicemesh/cmd-forwarder-vpp/blob/5a236699eac6e1db5856d702228d798370807425/internal/vppinit/links.go#L36) will try to process the missing interface, but when it does it will return without considering the remaining devices. (The outcome might be different among the running vpp forwarders.)
This might result in issues when for example trying to use Remote VLAN NSE with an interface that got ignored, hence the forwarder/vpp will not know about it.

IMHO an error during _InitLinks_ should either be considered fatal, or the code should filter out devices that does not exist on the host without skipping others.

So, basically the question is if a device selector file with unknown interfaces is considered an error or not.

**Comment by zolug** [comment]
Thanks, the PR seems to work perfectly.
